app.rb

```diff
@@ -1,19 +1,12 @@
-require './person'
-require './book'
```

Comment:
- [ ] Please look into your app. I cannot create a person (student or teacher). Kindly fix that.

  ![image](https://github.com/AlyAtalla/oop-library/assets/81468675/dea17c4a-d3b3-4d84-8859-cf5962d02152)

- [ ] Given that I could not create a person I could not verify if the data is well saved when a person is created. Kindly ensure that works as listed in the instructions.